<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xVFontDefinitionDlg</class>
 <widget class="QDialog" name="xVFontDefinitionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>define font</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="pExampleLab">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>1,2,3 Example Text ...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>example text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pExampleTxtLEdit">
       <property name="text">
        <string>1,2,3 Example Text ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="xVPropTable" name="pPropTable">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>25</number>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="pCancelTB">
       <property name="text">
        <string>&amp;cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pApplyTB">
       <property name="text">
        <string>&amp;apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xVPropTable</class>
   <extends>QTableWidget</extends>
   <header location="global">xvproptable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pCancelTB</sender>
   <signal>clicked()</signal>
   <receiver>xVFontDefinitionDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pApplyTB</sender>
   <signal>clicked()</signal>
   <receiver>xVFontDefinitionDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
